### DISCUSSION TITLE
[UX] - Allow traders to open the arbitration chat while the mediation chat is open

### DISCUSSION THREAD

**Discussion by huey735** [bisq-network/bisq#5227] [Ideas]
Traders often re-open the mediation ticket asking about how to proceed. Mediators have then to respond to them that they need to press "ctrl+o" once the mediation is closed.
It'd be great if after the DelayedPayoutTx is valid that the Traders had a way to move onto arbitration independently of the mediation ticket.

@jmacxx @leo816

**Comment by ghost** [comment]
Added it to the TODO list in #5207
